docs/guides/developer/docs/api/series-api.md

```diff
@@ -176,18 +244,33 @@ __Response__
 `204 (NO CONTENT)`: The series has been deleted.<br/>
 `404 (NOT FOUND)`: The specified series does not exist.
 
-# Metadata
 
-<!--- ##################################################################### -->
+## Metadata
+
+This section describes how to use the Application API to work with metadata catalogs associated to series.
+
+Opencast manages the bibliographic metadata of series using metadata catalogs which are identified by flavors.
+The default metadata catalog for Opencast series has the flavor `dublincore/series`. Opencast additionally supports
+extended metadata catalogs for series that can be configured.
+
+The Application API supports both the default series metadata catalog and series extended metadata catalogs.
+For the default series metadata catalog, the metadata is directly returned in the responses.
+
+Since the metadata catalogs can be configured, the Application API provides a facility to retrieve the catalog
+configuration of series metadata catalogs. For more details about this mechanism, please refer to
+["Metadata Catalogs"](types.md#metadata-catalogs).
+
 ### GET /api/series/{series_id}/metadata
 
 Returns a series' metadata of all types.
 
 __Response__
 
-`200 (OK)`: The series' metadata are returned.<br/>
+`200 (OK)`: The series' metadata are returned as [`catalogs`](types.md#catalogs)
```

Comment:
Returned as a list of catalogs?

Reply:
I've defined the type "catalogs" as being an array of catalog objects.